pwiz_tools/Skyline/Controls/SeqNode/SrmTreeNode.cs

```diff
@@ -439,6 +439,13 @@ public void ShowPickList(Point location, bool okOnDeactivate)
                 popup.Show(FormEx.GetParentForm(TreeView));
                 popup.Focus();
             }
+

```

Comment:
This blank line should be removed. There should never be white space between catch blocks.

Reply:
OK, done, thanks